app/services/push_notifications/send.rb

```diff
@@ -0,0 +1,24 @@
+module PushNotifications
+  class Send
+    def self.call(user:, title:, body:, payload:)
+      new(user: user, title: title, body: body, payload: payload).call
+    end
+
+    def initialize(user:, title:, body:, payload:)
+      @user = user
+      @title = title
+      @body = body
+      @payload = payload
+    end
+
+    def call
+      return unless FeatureFlag.enabled?(:mobile_notifications)
+
+      @user.devices.each do |device|
+        device.create_notification(@title, @body, @payload)
+      end
```

Comment:
**non blocking**: very unlikely a user has 100 devices but we might want to "prematurely optimize" here and use `.find_each`

Reply:
I don't see any reason NOT to optimize this now, thanks!